### PR TITLE
refactor: use coerce function of yargs to resolve path

### PR DESCRIPTION
**Pull request by shubheksha**
Removed the hack which was in place to do this & replaced it with the `coerce` function to resolve the path of the source directory. Solves #438

### CODE REVIEW

File: src/program.js [hunk 1]
```diff
@@ -154,9 +148,11 @@ Example: $0 --help run.
       alias: 's',
       describe: 'Web extension source directory.',
       default: process.cwd(),
-      normalize: true,
       requiresArg: true,
       type: 'string',
+      coerce: (arg) => {
```
Comment (rpl) [inline src/program.js]: it looks like this can be reduced to:

```
...
coerce: path.resolve,
...
```

**Comment by kumar303** [comment]
Thanks for the change. It feels much cleaner now 
le-happy

**Comment by atsay** [comment]
Thanks for contributing Shubheksha, and welcome to the community! I've added your merges on our [recognition page](https://wiki.mozilla.org/Add-ons/Contribute/Recognition#August_2016).

**Comment by shubheksha** [comment]
@atsay, thank you so much, this made my day! :smile: I really hope to stay around and contribute a lot more! 
There is one more [PR](https://github.com/mozilla/web-ext/pull/434) I opened which got merged, could you include this as well? 
Thanks again!

**Comment by atsay** [comment]
Done! Thanks for letting me know--we don't always catch everything, but we're working on ways to get better at it.

**Pull request merged**